indra/newview/app_settings/settings.xml

```diff
@@ -9376,7 +9376,7 @@
   <key>RenderMaxVRAMBudget</key>
   <map>
     <key>Comment</key>
-    <string>Maximum amount of texture memory to budget for (in MB), or 0 for autodetect.  Requires restart.</string>
+    <string>Maximum amount of texture memory to budget for (in MB), or 0 for autodetect. OS Windows Specific.</string>
```

Comment:
This is not correct. For the actual calculation where it really matters, it is the same for all OS. It is only OS specific for information purposes (lag meter floater and texture view console), where fake values are shown since that calculation is different.